gtfs/spec/en/reference.md

```diff
@@ -32,6 +33,22 @@ This section defines terms that are used throughout this document.
 * **Field optional** - The field column may be omitted from your feed. If you choose to include an optional column, each record in your feed must have a value for that column. You may include an empty string as a value for records that do not have values for the column. Some optional fields permit an empty string as a value. To enter an empty string, just omit any text between the commas for that field. Note that 0 is interpreted as "a string of value 0", and is not an empty string.
 * **Dataset unique** - The field contains a value that maps to a single distinct entity within the column. For example, if a route is assigned the ID **1A**, then no other route may use that route ID. However, you may assign the ID **1A** to a location because locations are a different type of entity than routes.
 
+## Field Types
+
+- **Ad-hoc** - Please see the detail column of the field for the definition of the field type.
+- **Color** - The field contains a color encoded as a six-character hexadecimal number. Refer to https://htmlcolorcodes.com/ to generate a valid value. E.g. "FFFFFF" for white, "000000" for black or "0039A6" for the A,C,E lines in NYMTA.
+- **Enum** - The field contains a value among a set of ad-hoc predefined constants defined in the "Details" column. E.g. the route_type field contains a "0" for tram, a "1" for subway...
+- **ID** - The field contains an identifier which uniquely identifies an entity (therefore  is dataset unique). Its value is not aimed to be displayed to the user, and is a sequence of any UTF-8 characters, but using only printable ASCII characters is recommended.
+- **Positive Integer** - The field contains a non-negative whole number (which includes 0).
+- **Language Code** - The field contains a IETF BCP 47 language code. For an introduction to IETF BCP 47, please refer to http://www.rfc-editor.org/rfc/bcp/bcp47.txt and http://www.w3.org/International/articles/language-tags/. E.g. "en" for English, "en-US" for American English or "de" for German.
+- **Latitude** - The field contains the WGS84 latitude as a float number (it includes decimals). The value must be higher and -90.0 and smaller than 90.0.
```

Comment:
For lat and long, I would change this to `a floating point (i.e., decimal) number`

Reply:
I would change to `The value must be greater than or equal to -90.0 and less than or equal to 90.0."

Reply:
Done. But I removed the (i.e.,decimal) which was cutting the definition in half and I got stuck at "what is a floating point?"

Reply:
I'm adding an example at the end.

Reply:
So technically this should actually read `The field contains the WGS84 latitude in decimal degrees`.

"decimal degrees" is the way the numeric decimal format is usually described (as opposed to degrees/minutes/seconds).

Reply:
updated